app/react/src/server/config/babel.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
         targets: {
           browsers: ['last 2 versions', 'safari >= 7'],
         },
-        modules: false,
+        modules: process.env.NODE_ENV === 'test' ? undefined : false,
```

Comment:
I can change it. Anyways, if they change the default it will be a major bump.

Want me to set to `commonjs`?